crates/rooch-framework/src/bindings/address_mapping.rs

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) RoochNetwork
```

Comment:
Consider removing the rust logic in the framework and put it in a separate crate, so that there is only move code in the framework?

Reply:
The `binding` is for binding a Rust code with the Move contract module. If we change the Move code, we also need to change the binding; I think putting them together is more suitable.